Dockerfile

```diff
@@ -1,31 +1,23 @@
-FROM centos:7
-
-RUN yum -y update && \
-    yum -y install java-1.8.0-openjdk
-
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.292.b10-1.el7_9.x86_64
-
-ENV JRE_HOME /usr/lib/jvm/jre
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
 RUN useradd -r -g root tomcat
 
-
-ADD  http://apache.cs.utah.edu/tomcat/tomcat-9/v9.0.46/bin/apache-tomcat-9.0.46.tar.gz /tmp
+ADD  https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.22/bin/apache-tomcat-9.0.22.tar.gz /tmp
 
 RUN cd /tmp && \
-    tar -xf apache-tomcat-9.0.46.tar.gz && \
-    mv apache-tomcat-9.0.46 /opt/tomcat/ && \
+    tar -xf apache-tomcat-9.0.22.tar.gz && \
+    rm apache-tomcat-9.0.22.tar.gz && \
+    mv apache-tomcat-9.0.22 /opt/tomcat/ && \
     chown -R tomcat: /opt/tomcat && \
-    sh -c 'chmod +x /opt/tomcat/bin/*.sh'
+    chmod +x /opt/tomcat/bin/*.sh
 
-ENV CATALINA_HOME /opt/tomcat/
-ENV CATALINA_PID /opt/tomcat/temp/tomcat.pid
+ENV CATALINA_HOME=/opt/tomcat/
+ENV CATALINA_PID=/opt/tomcat/temp/tomcat.pid
 ENV PATH=/opt/tomcat/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 # WORKDIR /opt/tomcat
 USER tomcat
 EXPOSE 8080
-CMD ["/bin/bash"]
-# CMD ["catalina.sh","run"]
```

Comment:
@naveen-armedia  this line is needed, right?  Without this I believe the container will immediately exit.

Reply:
Yes, @JonHolman updated now. Anyway, the docker images that inherit this base image will be using the same command for starting their application

Reply:
@JonHolman  This will be an image that is used to build other images on top of. I believe it will never be run on its own, so not having a CMD should not be a problem.

Reply:
@fabricetriboix  Sounds good.  I just wanted to be able to test the intermediate container.

Reply:
@JonHolman Yes, that's a good point, actually.